test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js

```diff
@@ -5,11 +5,20 @@ import { within } from '@testing-library/testcafe';
 const FORM_INFOBOX_ERROR_TITLE = '[data-se="o-form-error-container"] [data-se="callout"] > h3';
 const FORM_INFOBOX_ERROR_TITLE_V3 = '[data-se="callout"] > div > h2';
 const CANT_SCAN_BUTTON_TEXT = 'Setup without scanning a QR code.';
+const SETUP_OV_BUTTON_TEXT = 'Set up Okta Verify';
+const SETUP_ON_ANOTHER_MOBILE_DEVICE_TEXT = 'set up Okta Verify on another mobile device';
+const GO_BACK_LINK_TEXT = 'Go back';
 const FORM_SELECTOR = '[data-se="o-form-explain"]';
 const SUB_HEADER = '[data-se="subheader"]';
 const COPY_ORG_LINK_BUTTON_CLASS = '.copy-org-clipboard-button';
 const CLIPBOARD_TEXT = 'data-clipboard-text';
 const DOWNLOAD_OV_LINK_CLASS = '.download-ov-link';
+const APP_STORE_LINK_CLASS = '.app-store-link';
+const APP_STORE_LOGO_CLASS = '.app-store-logo';
```

Comment:
After removing `.app-store-logo` you can switch this selector to be `[data-se='app-store-logo']`